code/mob/transform_procs.dm

```diff
@@ -776,10 +776,10 @@ var/list/antag_respawn_critter_types =  list(/mob/living/critter/small_animal/fl
 	// 	newbody.abilityHolder.transferOwnership(newbody)
 	// src.abilityHolder = null
 
+	// There are some traits removed in the afterlife bar: see proc/traitHolder/addTrait()
 
 	newbody.UpdateOverlays(image('icons/misc/32x64.dmi',"halo"), "halo")
 	newbody.set_clothing_icon_dirty()
-	newbody.set_loc(target_turf)
```

Comment:
Are we sure this isn't necessary in some edge cases?

Reply:
I've tested this with stowaway multiple times and it doesn't seem to affect anything. Should be the same for pilot and heavy sleeper
The reason i do this is that if we only set the location after the fact, addTrait won't know this human is spawning in the afterlife bar

Reply:
Yea I'm aware. But a paranoid check of the form `if(newbody.loc != target_turf) newbody.set_loc(target_turf)` can't hurt.
But also if you say it's ok then it's probably ok.